app/src/org/commcare/dalvik/activities/EntitySelectActivity.java

```diff
@@ -785,6 +786,7 @@ private void select() {
         // create intent for return and store path
         Intent i = new Intent(EntitySelectActivity.this.getIntent());
         i.putExtra(SessionFrame.STATE_DATUM_VAL, selectedIntent.getStringExtra(SessionFrame.STATE_DATUM_VAL));
+        ACRAUtil.addCustomData(ACRAUtil.CURRENT_CASE, selectedIntent.getStringExtra(SessionFrame.STATE_DATUM_VAL));

```

Comment:
I'm a bit mixed on whether it's OK for us to submit Case Id's to ACRA's stuff unless we're convinced we have a full chain of custody that's acceptable. We actually host our own server for htis data, right?